install-pcan.md

```diff
@@ -0,0 +1,20 @@
+# Install PCAN-M.2 Driver for Linux (Ubuntu 16.04)
+
+Official page:  [link](https://www.peak-system.com/fileadmin/media/linux/index.htm)
+Manual:         [link](https://www.peak-system.com/fileadmin/media/linux/files/PCAN-Driver-Linux_UserMan_eng.pdf)
+
+```bash
+sudo apt-get install libpopt-dev libelf-dev
+cd && mkdir -p ~/src/ && mkdir -p ~/src/peak && cd ~/src/peak
+wget https://www.peak-system.com/fileadmin/media/linux/files/peak-linux-driver-8.5.1.tar.gz
+tar -xzf peak-linux-driver-8.5.1.tar.gz # Untar the compressed tarball file
+cd peak-linux-driver-8.5.1/ 
+make -C driver PCI=PCI_SUPPORT DNG=NO_DONGLE_SUPPORT USB=NO_USB_SUPPORT ISA=NO_ISA_SUPPORT PCC=NO_PCCARD_SUPPORT
+sudo make -C driver install
+make -C test && sudo make -C test install # test utilities (optional)
+sudo modprobe pcan # driver loading
+cd && cd repos/yarp-devices/build/
```

Comment:
I'd rather split this long code block and start a new one at this very line. By the way, said line assumes that yarp-devices has been cloned, configured and compiled, therefore please add a reference to [yarp-devices-install.md](https://github.com/roboticslab-uc3m/yarp-devices/blob/develop/doc/yarp-devices-install.md) before this new block.